packages/firebase/index.d.ts

```diff
@@ -1220,6 +1234,39 @@ declare namespace firebase.app {
   }
 }
 
+/**
+ * @webonly
+ */
+declare namespace firebase.installations {
+  /**
+   * The Firebase Installations service interface.
+   *
+   * Do not call this constructor directly. Instead, use
+   * {@link firebase.installations `firebase.installations()`}.
+   */
+  export interface Installations {
+    /**
+     * Creates a Firebase Installation if there isn't one for the app and
+     * returns the Installation ID.
+     *
+     * @return Firebase Installation ID
+     */
+    getId(): Promise<string>;
+
+    /**
+     * Returns an Authentication Token for the current Firebase Installation.
+     *
+     * @return Firebase Installation Authentication Token
+     */
+    getToken(): Promise<string>;
```

Comment:
Why is this not called #getAuthToken()?

The term "token" is heavily overloaded in the space of Firebase and particularly with the background of Instance-ID, where a "token" referred to an FCM-Token, which is an FCM-address that can be used to send notifications to a Firebase Installation.
This token will not be able to do that, but is a dedicated authentication-token.
Thus, I would strongly recommend to add this distinction to the method name for clarification.